### PR TITLE
Add Mailcatcher

### PR DESCRIPTION
**Pull request by allenmoore**
MailCatcher is one of the tools built into VVV, which allows for local email testing, without storing email credentials in a database. I can't speak for everyone, but I've utilized MailCatcher quite a bit for multiple projects for testing HTML emails, custom user notifications, and more.

This pull request adds a new container for MailCatcher, using this container -- https://store.docker.com/community/images/schickling/mailcatcher. I've also added the necessary `sendmail_path` settings to the `php.ini` file and updated the `readme.md` file to include documentation for MailCatcher.

### CODE REVIEW

File: config/php-fpm/php.ini [hunk 1]
```diff
@@ -2,3 +2,6 @@
 
 upload_max_filesize = 100M
 post_max_size = 100M
+
+[mail function]
+sendmail_path = /usr/bin/env catchmail -smtp-port 1025 -f allen.moore@10up.com
```
Comment (dwainm) [inline config/php-fpm/php.ini:7]: @allenmoore I see you've added your email as an argument. Will the setup work with a generic/dummy email address?

**Comment by tlovett1** [comment]
@allenmoore there are two mailcatcher PRs. Have you looked at the other one?

**Comment by dwainm** [comment]
@tlovett1 have you been able to get the other one to work? This one's not working for me. Trying the other one now.

**Comment by tlovett1** [comment]
@allenmoore merged your PR then pushed a bunch of changes to make it work. Can you test? @dwainm

**Comment by mikedoubintchik** [comment]
I'm attempting to setup mailcatcher with traefik routing. I'm trying to understand how to properly configure the docker-compose.yml file. What I have currently is giving me an Internal Server Error (below). 

Thanks in advance!

```
  ## Mailcatcher
  mailcatcher:
    image: schickling/mailcatcher
    restart: always
    environment:
      MAILCATCHER_PORT: 1025
    labels:
      - 'traefik.backend=mailcatcher'
      - 'traefik.port=1080'
      - 'traefik.frontend.rule=Host:mailcatcher.wp.docker.localhost'
```

**Pull request merged**